fincaguarumo/src/components/icons/Error.tsx

```
@@ -0,0 +1,14 @@
+import React from "react"
+
+const Error = ({ ...props }) => {
```

Biome (2.1.2)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)